demos/python_demos/whiteboard_inpainting_demo/whiteboard_inpainting_demo.py

```diff
@@ -0,0 +1,184 @@
+    parser.add_argument('-i', type=str, help='Input sources (index of camera \
+                        or path to a video file)', required=True)
+    parser.add_argument('-mi', '--m_instance_segmentation', type=str, required=False,
+                        help='Path to the instance segmentation model')
+    parser.add_argument('-ms', '--m_semantic_segmentation', type=str, required=False,
+                        help='Path to the semantic segmentation model')
+    parser.add_argument('-t', '--threshold', type=float, default=0.6,
+                        help='Threshold for person instance segmentation model')
+    parser.add_argument('--output_video', type=str, default='', required=False,
+                        help='Optional. Path to output video')
+    parser.add_argument("--no_show", help="Optional. Don't show output", action='store_true')
+
+    parser.add_argument('-d', '--device', type=str, default='CPU')
+    parser.add_argument('-l', '--cpu_extension', type=str, default=None,
+                        help='MKLDNN (CPU)-targeted custom layers.Absolute \
+                              path to a shared library with the kernels impl.')
+    args = parser.parse_args()
+
+    if not path_exist(args.i, type='file') and not str.isdigit(args.i):
+        raise ValueError('File not found or wrong camera id: {}'.format(args.i))
+    else:
+        capture = VideoCapture(args.i)
+
+    if (args.m_instance_segmentation and args.m_semantic_segmentation) or \
+       (not args.m_instance_segmentation and not args.m_semantic_segmentation):
+        raise ValueError('Set up exectly one of segmentation models: '\
+                         '--m_instance_segmentation or --m_semantic_segmentation')
+
+    frame_size, fps = capture.get_source_parameters()
+    out_frame_size = (int(frame_size[0]), int(frame_size[1] * 2))
+
+    mouse = MouseClick()
+    cv2.namedWindow(WINNAME)
+    cv2.setMouseCallback(WINNAME, mouse.get_points)
+
+    if args.output_video:
+        fourcc = cv2.VideoWriter_fourcc(*'XVID')
+        output_video = cv2.VideoWriter(args.output_video, fourcc, fps, out_frame_size)
+    else:
+        output_video = None
+
+    log.info("Creating Inference Engine")
+    ie = IECore()
+    if args.m_instance_segmentation:
+        segmentation = MaskRCNN(ie, args.m_instance_segmentation, args.threshold,
+                                args.device, args.cpu_extension, 1)
+    elif args.m_semantic_segmentation:
+        segmentation = SemanticSegmentation(ie, args.m_semantic_segmentation, args.threshold,
+                                            args.device, args.cpu_extension)
+
+    has_frame = True
+    output_frame = None
+    black_board = False
+    frame_number = 0
+    key = -1
+
+    while has_frame:
+        start = time.time()
+        if not args.no_show:
+            key = check_pressed_keys(key)
+            if key == 27:
+                break
+            elif key == 105:
```

Comment:
What is `105`?

Reply:
It is an 'i' key. Fixed and added a comment.